.github/workflows/publish-qemu-bin.yml

```diff
@@ -0,0 +1,163 @@
+name: Publish Qemu External Dependency
+
+on:
+  workflow_dispatch:
+  release:
+    types: [published]
+  pull_request:
+    branches:
+      - main
+    paths:
+      - .github/workflows/publish-qemu-bin.yml
+      - .github/publish-qemu-config.yml
+
+env:
+  QEMU_VERSION_FILE: .github/publish-qemu-config.yml
+
+jobs:
+  qemu-windows:
+    name: Build Qemu for Windows
+    runs-on: windows-latest
+  
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+    - name: Get current Qemu version
+      id: qemu-version
+      run: |
+        $qemuVersion = (Get-Content -Path $env:QEMU_VERSION_FILE | ConvertFrom-Json).windows
+        "QEMU_VERSION=$qemuVersion" | Out-File -FilePath "$env:GITHUB_OUTPUT" -Append
+      shell: pwsh
+    - name: Download Qemu v${{ steps.qemu-version.outputs.QEMU_VERSION }}
+      env:
+        QEMU_VERSION: ${{ steps.qemu-version.outputs.QEMU_VERSION }}
+      run: |
+        choco install qemu --version=$env:QEMU_VERSION
+    - name: Stage Qemu Binaries

```

Comment:
Did you check if any license files need to be included for any of the binary components?

Kind of like the [license file](https://github.com/microsoft/mu_crypto_release/blob/release/202302/CryptoBinPkg/Driver/Packaging/License.txt) included in shared crypto releases.

Reply:
We now upload

- https://github.com/qemu/qemu/blob/master/COPYING
- https://github.com/qemu/qemu/blob/master/LICENSE

Using the specific tag we are downloading - not master.